qui/utils.py

```diff
@@ -80,11 +80,25 @@ def run_asyncio_and_show_errors(loop, tasks, name, restart=True):
             exit_code = 1
     return exit_code
 
+def check_update(vm) -> bool:
```

Comment:
Everywhere this and check_support is used, they are wrapped in catching exceptions. I wonder if it wouldn't be neater to get this exception-catching done here, with some good defaults (no updates available, still supported?)

Reply:
> Everywhere this and check_support is used, they are wrapped in catching exceptions. I wonder if it wouldn't be neater to get this exception-catching done here, with some good defaults (no updates available, still supported?)

Honestly I do not remember why I did this. It was during the (relatively big & complex) global clipboard configurable size and `.metadata` project. And that one consumed a lot of brain power. I am going to move the catching exceptions inside them now. But I will make it a new commit in this PR for easier follow-up.

Reply:
This is done